client/src/css/core/layout.css

```
@@ -345,6 +345,16 @@
   padding: rem(6) 1rem;
 }
 
+/* Video Image Label */
+@mixin video-label {
+  background-color: var(--orange);
+  background-image: url("data:image/svg+xml,%3Csvg width='16' height='17' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M.7 17a.694.694 0 01-.495-.207.713.713 0 01-.205-.501V.708a.716.716 0 01.343-.61.692.692 0 01.693-.01l13.983 7.791a.703.703 0 01.362.621.716.716 0 01-.362.62L1.036 16.913A.692.692 0 01.699 17z' fill='%23fff'/%3E%3C/svg%3E");
```

Could reduce this a bit
```suggestion
  background-image: url("data:image/svg+xml,%3Csvg fill='none'%3E%3Cpath d='M.7 17a.694.694 0 01-.495-.207.713.713 0 01-.205-.501V.708a.716.716 0 01.343-.61.692.692 0 01.693-.01l13.983 7.791a.703.703 0 01.362.621.716.716 0 01-.362.62L1.036 16.913A.692.692 0 01.699 17z' fill='%23fff'/%3E%3C/svg%3E");
```